src/tz390.java

```diff
@@ -3841,6 +3875,25 @@ public void create_opcodes()  // Routine added for RPI 1209
          process_opcodes(op_table_ZS3);
          process_opcodes(op_table_ZS4);
          process_opcodes(op_table_Z15);  // rpi 2202
+         process_opcodes(op_table_z16); // DSH #414
```

Comment:
z16 opcodes should not be included when machine level is z15

Reply:
fixed.